src/gluonnlp/model/bert.py

```diff
@@ -534,9 +553,28 @@ def _decode(self, sequence, masked_positions):
     'word_embed': None,
 }
 
+ernie_12_768_12_hparams = {
+    'attention_cell': 'multi_head',
+    'num_layers': 12,
+    'units': 768,
+    'hidden_size': 3072,
+    'max_length': 513,
```

Comment:
Are you aware why they increase from 512 to 513?

Reply:
Not aware why they did so.